### PR TITLE
[OCLC++][vload] Remove address spase qualifier from return type

### PR DESCRIPTION
**Pull request by kkyzylov**

### CODE REVIEW

File: include/opencl_vector_load_store [hunk 1]
```diff
@@ -110,7 +110,7 @@ template <size_t N, class T>
 __ALWAYS_INLINE enable_if_t<!is_same<make_vector_t<T, N>, T>::value, make_vector_t<T, N>>
 vload(size_t offset, const T* ptr)
 {
-    return __spirv::__make_vloadn_call<make_vector_t<T,N>>(offset, ptr, N);
+    return __spirv::__make_vloadn_call<make_vector_t<remove_as_t<T>,N>>(offset, ptr, N);
```
Comment (AlexeySotkin) [inline include/opencl_vector_load_store]: Shouldn't the next function be updated as well ?

**Comment by AlexeySotkin** [comment]
Does the bug which you have fixed, affect other functions in this file ?

**Comment by kkyzylov** [comment]
No.

**Comment by kkyzylov** [comment]
No.

**Comment by bsochack** [comment]
Looks good to me.

**Comment by AlexeySotkin** [comment]
Looks good to me, too.

**Pull request merged**